conjure-java-core/src/test/java/com/palantir/conjure/java/JerseyServiceGeneratorTests.java

```diff
@@ -97,6 +97,25 @@ public void testBinaryReturnResponse() throws IOException {
         validateGeneratorOutput(files, Paths.get("src/test/resources/test/api"), ".jersey.binary_as_response");
     }
 
+    @Test
+    public void testSortedMethodParameters() throws IOException {
+        ConjureDefinition def = Conjure.parse(
+                ImmutableList.of(new File("src/test/resources/example-param-sorting.yml")));
+        List<Path> files = new JerseyServiceGenerator(Collections.emptySet())
+                .emit(def, folder.getRoot());
+        validateGeneratorOutput(files, Paths.get("src/test/resources/test/api"), ".jersey.sorted_params");
```

Comment:
Aren't there existing example service methods which we can reuse without a new conjure yml definition? The default `.jersey` outputs should be sufficient to test this case, I don't think this test adds anything